jobs/build/olm_bundle/Jenkinsfile

```diff
@@ -0,0 +1,125 @@
+node {
+    checkout scm
+    olm_bundles = load('olm_bundles.groovy')
+}
+
+pipeline {
+    agent any
+
+    options {
+        disableResume()
+        skipDefaultCheckout()
+    }
+
+    parameters {
+        choice(
+            name: 'BUILD_VERSION',
+            choices: olm_bundles.commonlib.ocpVersions,
+            description: 'OCP Version'
+        )
+        string(
+            name: 'ONLY',
+            description: '(Optional) List **only** the operators you want '  +
+                         'to build, everything else gets ignored.\n'         +
+                         'Format: Comma and/or space separated list of brew '+
+                         'packages (e.g.: cluster-nfd-operator-container)\n' +
+                         'Leave empty to build all (except EXCLUDE, if defined)',
+            defaultValue: ''
+        )
+        string(
+            name: 'EXCLUDE',
+            description: '(Optional) List the operators you **don\'t** want ' +
+                         'to build, everything else gets built.\n'            +
+                         'Format: Comma and/or space separated list of brew ' +
+                         'packages (e.g.: cluster-nfd-operator-container)\n'  +
+                         'Leave empty to build all (or ONLY, if defined)',
+            defaultValue: 'cluster-nfd-operator-container'
+            /* Temporarily excluding cluster-nfd-operator-container, until
+               https://github.com/openshift/cluster-nfd-operator/issues/82
+               gets solved */
+        )
+        string(
+            name: 'OLM_OPERATOR_ADVISORY',
+            description: '(Optional) Fetch OLM Operators NVRs from advisory.\n' +
+                         'Leave empty to fetch NVRs from "brew latest-build"',
+            defaultValue: ''
+        )
+        string(
+            name: 'BUNDLES_ADVISORY',
```

Comment:
I don't think we'll want a separate advisory for bundles. We'll just put them in the same advisory the rest of extras go in.

I can't think of a reason we'd want to run this job and not attach the results anywhere (other than... testing the job?).

Maybe the default could be to attach them to current extras, and this parameter could be to explicitly send them somewhere else (e.g. rebuilding something for an in-flight-but-not-latest release that wasn't right initially).